glue_astronomy/translators/tests/test_trace.py

```diff
@@ -0,0 +1,19 @@
+from specreduce import tracing
+from specreduce.utils.synth_data import make_2dspec_image
+
+from glue.core import Data, DataCollection
+
+
+def test_trace():
+
+    image = make_2dspec_image()
+    trace = tracing.FlatTrace(image, 5)
+
+    data_collection = DataCollection()
+
+    data_collection['trace'] = trace
+    data = data_collection['trace']
+    assert isinstance(data, Data)
```

Comment:
```suggestion
    assert isinstance(data, Data)
    assert data['y'] == trace.trace
```
Hope I got that right, not sure if it's worth also checking the (shape of) `data['x']`